docs/command_line_tool/phylum_project.md

```diff
@@ -4,7 +4,7 @@ category: 6255e67693d5200013b1fa3e
 hidden: false
 ---
 
-Create, list, link and set thresholds for projects
+Create, list, and link projects
```

Comment:
```suggestion
Create, delete, list, and link projects
```

Reply:
I think this just demonstrates that this documentation isn't really maintainable. I've changed it to something more generic that shouldn't require frequent updates.